### PR TITLE
Wiki Update 3

### PR DESCRIPTION
**Pull request by xNickyDev**
Files from h-i
- Changed headers "Usage" to "Syntax"
- Added missing examples
- Split [`$hostingExpireTime`](https://github.com/NilPointer-Software/bdfd-wiki/blob/dev/src/bdscript/hostingExpireTime.md) into two files -> [`$hostingExpireTime[]`](https://github.com/NilPointer-Software/bdfd-wiki/blob/dev/src/bdscript/hostingExpireTimeComplex.md)
- Other small changes

### CODE REVIEW

**Comment by MineBartekSA** [comment]
Sorry for the wait, I'll review this asap in the coming days

**Comment by Rainb0wKey** [comment]
`$hostingExpireTime[Return unix timestamp]` > `$hostingExpireTime[Return unix timestamp?]`

File: src/bdscript/hostingExpireTime.md [hunk 1]
```diff
@@ -1,25 +1,14 @@
 # $hostingExpireTime 
-Returns expire time for the hosting 
+Returns expire time for the hosting.
```
Comment (MineBartekSA) [inline src/bdscript/hostingExpireTime.md]:
``` suggestion
Returns your bot's hosting expiration date.
```

File: src/bdscript/hostingExpireTimeComplex.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# $hostingExpireTime[]
+Returns expire time for the hosting. If "yes" is provided, the function returns the expire time in UNIX Timestamp.
```
Comment (MineBartekSA) [inline src/bdscript/hostingExpireTimeComplex.md]:
``` suggestion
Returns your bot's hosting expiration date. If "yes" is provided, the function returns the expiration date in a UNIX timestamp.
```

File: src/bdscript/hostingExpireTimeComplex.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+## Syntax
+``` 
+$hostingExpireTime[Return unix timestamp?]
+``` 
+
+### Parameters
+- `Return unix timestamp?` `(Type: Bool || Flag: Required)`: Whether to return the expiration time in UNIX Timestamp or not.
```
Comment (MineBartekSA) [inline src/bdscript/hostingExpireTimeComplex.md]:
``` suggestion
- `Return unix timestamp?` `(Type: Bool || Flag: Required)`: Whether to return the expiration date in a UNIX timestamp or not.
```

**Review by MineBartekSA — CHANGES_REQUESTED** [review]

**Review by MineBartekSA — APPROVED** [review]
Thank you for your hard work!

**Pull request merged**